website/docs/r/registry_module.html.markdown

```diff
@@ -108,8 +108,8 @@ The `vcs_repo` block supports:
 * `identifier` - (Required) A reference to your VCS repository in the format
   `<organization>/<repository>` where `<organization>` and `<repository>` refer to the organization (or project key, for Bitbucket Server)
   and repository in your VCS provider. The format for Azure DevOps is <organization>/<project>/_git/<repository>.
-* `oauth_token_id` - (Required) Token ID of the VCS Connection (OAuth Connection Token)
-  to use.
+* `oauth_token_id` - (Required) Token ID of the VCS Connection (OAuth Connection Token) to use. This conflicts with `github_app_installation_id` and can only be used if `github_app_installation_id` is not used.
```

Comment:
same here, the `oauth_token_id` and `github_app_installation_id` are now optional